cron/rss_reader.py

```diff
@@ -10,11 +10,12 @@ def __init__(self, feed_url):
     def read_feed(self, newer_than):
         feed = feedparser.parse(self.feed_url)
 
+        print(len(feed.entries))
```

Comment:
This `print` is probably not too helpful to keep around?

Reply:
Yeah, I was using it while debugging and missed to remove it